tgs/src/main/cpp/tgs/Statistics/Random.cpp

```diff
@@ -72,12 +80,27 @@ namespace Tgs
 
   int Random::generateInt()
   {
-    return rand();
+#if defined(NEW_RAND)
+    return (*_rnd)();
+#else
+    if (_is_single)
+      return rand();
+    else
+      return rand_r(&_seed);
+#endif
   }
 
   void Random::seed(unsigned int s)
   {
-    srand(s);
+#if defined(NEW_RAND)
```

Comment:
Yeah, I debated the usefulness of leaving it in there for future consideration but I think we should remove it.  I'll do that right now.